com.unity.multiplayer.mlapi/Editor/NetworkingManagerEditor.cs

```diff
@@ -57,6 +57,29 @@ public class NetworkingManagerEditor : Editor
     private readonly List<Type> transportTypes = new List<Type>();
     private string[] transportNames = new string[] { "Select transport..." };
 
+    /// <summary>
+    /// Helper method that gets either the server or non-server NetworkingManager in the scene, or null if not present. 
+    /// </summary>
+    public static NetworkingManager GetNetworkingManager(bool isServer)
+    {
+        foreach (var manager in FindObjectsOfType<NetworkingManager>())
+        {
+            if (manager.IsServer == isServer)
```

Comment:
So I am confused by the IsServer check because from what I can tell its always false? https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/495/files#diff-641f8f5b259017aefa68498910442cb599d894ed91c96e4811d0a79b5f0151feR24

Reply:
no, that's for the `NetworkedObjectEditor.cs` in the Editor, not at Runtime I believe, no?